SU2_CFD/src/numerics/NEMO/convection/msw.cpp

```diff
@@ -198,7 +198,8 @@ CNumerics::ResidualType<> CUpwMSW_NEMO::ComputeResidual(const CConfig *config) {
                                                       epsilon*epsilon));
 
   /*--- Compute projected P, invP, and Lambda ---*/
-  CreateBasis(UnitNormal);
+  su2double l[MAXNDIM], m[MAXNDIM];
```

Comment:
thanks for untangling some of this stuff.  Its on the list to go through and completely abstract away....eventually.